index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
```

Comment:
I suggest that if you are going to link a file with the styles, don't do ⛔  if it is empty. 
![image](https://user-images.githubusercontent.com/110146573/221637467-67897125-a220-4cde-8e67-58afdafa3135.png)